server/routes/overview/mappers/functionalSkillsMapper.test.ts

```diff
@@ -66,39 +68,45 @@ describe('functionalSkillsMapper', () => {
     }
 
     // When
-    const actual = toFunctionalSkills(learnerProfiles)
+    const actual = toFunctionalSkills(learnerProfiles, prisonNumber)
```

Comment:
The mapper has been changed (next file down 👇 ) to include the prison number in the mapping